panel/src/components/Views/ModelView.vue

```diff
@@ -32,40 +34,28 @@ export default {
 		tabs: {
 			type: Array,
 			default: () => []
-		}
+		},
+		uuid: String
 	},
 	computed: {
-		content() {
-			return this.$panel.content.values;
+		changes() {
+			return this.$panel.content.changes;
 		},
-		id() {
-			return this.model.link;
+		hasTabs() {
+			return this.tabs.length > 1;
 		},
 		isLocked() {
-			return this.$panel.content.isLocked;
+			return false;
 		},
 		isUnsaved() {
-			return this.$panel.content.hasUnpublishedChanges;
+			return this.$panel.content.hasChanges;
 		},
 		protectedFields() {
 			return [];
 		}
 	},
-	watch: {
-		"$panel.view.timestamp": {
-			handler() {
-				this.$store.dispatch("content/create", {
-					id: this.id,
-					api: this.id,
-					content: this.model.content,
-					ignore: this.protectedFields
-				});
-			},
-			immediate: true
-		}
-	},
 	mounted() {
-		this.onInput = debounce(this.onInput, 50);
+		this.autosave = debounce(this.autosave, 200);
```

Comment:
`Thought:` Is pure debouncing the best way? If I keep continuously writing to a field, could be that there is no save for minutes, right? Do we need a "debounce but ensure at least every x seconds"?

Reply:
I think we need to tackle this one separately. We would probably need to start a timeout on every autosave call and then trigger another call a 1 minute later or so. This way, even if you don't type anything, you would still keep on firing new autosave events at least every minute.